src/Node/ElseClauseNode.php

```diff
@@ -14,6 +14,6 @@ class ElseClauseNode extends Node {
     public $elseKeyword;
     /** @var Token */
     public $colon;
-    /**@var StatementNode | StatementNode[] */
+    /** @var StatementNode | StatementNode[] */
```

Comment:
It does appear to be wrong, at least it isn't parsed by phpDocumentor when it's space-separated. There are a ton of those in the parser, I'll fix them all at once. Also some in the language server.